### ISSUE TITLE
date_time_picker doesn't work with custom pages

### ISSUE THREAD

**Issue by mabdelfattah** [platanus/activeadmin_addons#84]
I want to use date_time_picker in `active_admin_form_for` in `ActiveAdmin.register_page` but it shows the following error:
`undefined method 'input_name' for nil:NilClass`

**Comment by ldlsegovia** [comment]
It worked for me with `active_admin_form_for`

**/src/fanticket/app/admin/category.rb**

```ruby
ActiveAdmin.register Category do
  form partial: 'form'
end
```

**/myapp/app/views/admin/categories/_form.html.erb**

```ruby
<%= active_admin_form_for(resource, url: admin_category_url(resource)) do |f|
  f.inputs do
    f.input :name
    f.input :created_at, as: :date_time_picker
  end
  actions
end
%>
```

<img width="602" alt="screen shot 2016-11-17 at 12 44 48 pm" src="https://cloud.githubusercontent.com/assets/3026413/20396051/af9eaf5e-acc3-11e6-8fa9-d0b984631a78.png">

**Comment by mabdelfattah** [comment]
No, it is not working on 'register_page'.

**Comment by mabdelfattah** [comment]
Fixed here: #97
